mesos/jenkins/marathon-remote.json.tmpl

```diff
@@ -0,0 +1,88 @@
+{
+  "id": "/jenkins/master",
+  "cmd": null,
+  "cpus": 2,
+  "mem": 4096,
+  "disk": 0,
+  "instances": 1,
+  "constraints": [
+    [
+      "hostname",
+      "CLUSTER",
+      "10.0.24.35"
```

Comment:
If this is really a template file, hard references here should be replaced with `{{host_ip}}` or similar.